db/models/account.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const {
+    Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+    class Account extends Model {
+        /**
+         * Helper method for defining associations.
+         * This method is not a part of Sequelize lifecycle.
+         * The `models/index` file will call this method automatically.
+         */
+
+    }
+    Account.init({
+        name: DataTypes.STRING,
+        number: DataTypes.STRING,
```

Comment:
please, add a currency field to this model
also maybe we need to add the color of the card to store it here

Reply:
Here is still 3 fields. Please, add a currency and color fields to this model
Also, we need account type (card, cash, and stash)